ansible_ai_connect/ai/api/telemetry/schema2_utils.py

```diff
@@ -0,0 +1,46 @@
+#  Copyright Red Hat
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from ansible_ai_connect.ai.api.telemetry.schema2 import (
+    AnalyticsTelemetryEvents,
+    OneClickTrialPlan,
+    OneClickTrialStarted,
+)
+from ansible_ai_connect.ai.api.utils.segment_analytics_telemetry import (
+    send_segment_analytics_event,
+)
+
+
+def oneclick_trial_event_send(user):
+    send_segment_analytics_event(
+        AnalyticsTelemetryEvents.ONECLICK_TRIAL_STARTED,
+        lambda: _oneclick_trial_event_build(user),
+        user,
+    )
+
+
+def _oneclick_trial_event_build(user):
+    return OneClickTrialStarted(
+        plans=[
+            OneClickTrialPlan(
+                created_at=plan.created_at,
+                expired_at=plan.expired_at,
+                is_active=plan.is_active,
+                name=plan.plan.name,
+                id=plan.plan.id,
+            )
+            for plan in user.userplan_set.all()
+        ],
+        rh_user_org_id=None if user.organization is None else user.organization.id,
```

Comment:
`rh_user_org_id` cannot be `None`. We should raise a `ValueError` exception in this case.

Reply:
done